src/m365/pp/commands/managementapp/managementapp-list.ts

```diff
@@ -0,0 +1,36 @@
+import { Logger } from '../../../../cli';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+class PpManagementAppListCommand extends GraphCommand {
+  public get name(): string {
+    return commands.MANAGEMENTAPP_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists management applications for Power Platform';
+  }
+
+  public commandAction(logger: Logger, args: any, cb: () => void): void {
+    const requestOptions: any = {
+      // This should be refactored once we implement a PowerPlatform base class as api.bap will differ between envs.
+      url: `https://api.bap.microsoft.com/providers/Microsoft.BusinessAppPlatform/adminApplications?api-version=2020-06-01`,
```

Comment:
Does this single request return all items, or do we need to consider paging and we should use the `getAllItems` pattern instead?

Reply:
Maybe you'll get some hints if you set the request metadata to verbose